lib/cli_interface.rb

```diff
@@ -1,68 +1,71 @@
+require 'yaml'
+
 class CliInterface
   attr_accessor :input
   attr_accessor :output
 
   def initialize(input, output)
     @input = input
     @output = output
+    @file = YAML::load(File.open('en_text.yml'))
   end
 
   def display_board(board)
     board.each { |line| display_line(line) }
   end
 
   def next_move(mark)
-    output.print("#{mark} what is your next move? ")
+    output.print("#{mark} #{@file['next_move']} ")
     input.gets
   end
 
   def type_game
-    output.print("Which game do you want?\n")
+    output.puts("#{@file['which_game']}")
```

Comment:
There is a lot of duplication amongst the methods in this file. Can you extract out the `output.puts('some_key')` into a method then just call it in each one. Eg: `display('which_game')    display('not_free')`